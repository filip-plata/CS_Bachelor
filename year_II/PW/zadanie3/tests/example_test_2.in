6 3 4
1 x[0] = (x[1] + 1)
2 x[1] = (x[2] + 1)
3 x[2] = 1
4 x[0] 0
5
6 x[1] 2
