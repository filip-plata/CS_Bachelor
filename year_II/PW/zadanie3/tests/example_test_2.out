1 P
2 P
3 P
4 P 0
5 P 3
6 P 3
